webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { compose } from 'redux';
+import { Button } from 'patternfly-react';
+
+import { getParams } from '../../../common/urlHelpers';
+import PageLayout from '../../../pages/common/PageLayout/PageLayout';
+import ModelsTable from '../../../components/ModelsTable';
+import { withRenderHandler, withPageLayout } from '../../../common/HOC';
+import EmptyModelsPage from './EmptyModelsPage';
+
+import searchProps from '../consts';
+
+const layoutProps = {
```

Comment:
whats happening when I search? Turbolinks?
We should trigger an API call whenever the query changes, and update the table. its easy because we can handle the history stack. I implemented it all in #6733